app/models/landable/access_token.rb

```diff
@@ -5,6 +5,7 @@ class AccessToken < ActiveRecord::Base
     belongs_to :author
     validates_presence_of :author_id
     validates_presence_of :expires_at
+    validates_presence_of :permissions
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.